Gem/Code/Source/MultiplayerSampleAWSGameLiftServerSystemComponent.cpp

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ * 
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "MultiplayerSampleAWSGameLiftServerSystemComponent.h"
+
+#include <AzCore/Serialization/SerializeContext.h>
+#include <AzCore/Serialization/EditContext.h>
+#include <AzCore/Serialization/EditContextConstants.inl>
+
+#include <Request/AWSGameLiftServerRequestBus.h>
+
+namespace MultiplayerSample
+{
+    void MultiplayerSampleAWSGameLiftServerSystemComponent::Reflect(AZ::ReflectContext* context)
+    {
+        if (AZ::SerializeContext* serialize = azrtti_cast<AZ::SerializeContext*>(context))
+        {
+            serialize->Class<MultiplayerSampleAWSGameLiftServerSystemComponent, AZ::Component>()
+                ->Version(0)
+                ;
+        }
+    }
+
+    void MultiplayerSampleAWSGameLiftServerSystemComponent::GetProvidedServices(AZ::ComponentDescriptor::DependencyArrayType& provided)
+    {
+        provided.push_back(AZ_CRC_CE("MultiplayerSampleAWSGameLiftServerService"));
+    }
+
+    void MultiplayerSampleAWSGameLiftServerSystemComponent::GetIncompatibleServices(AZ::ComponentDescriptor::DependencyArrayType& incompatible)
+    {
+        incompatible.push_back(AZ_CRC_CE("MultiplayerSampleAWSGameLiftServerService"));
+    }
+
+    void MultiplayerSampleAWSGameLiftServerSystemComponent::GetRequiredServices(AZ::ComponentDescriptor::DependencyArrayType& required)
+    {
+        required.push_back(AZ_CRC_CE("NetworkingService"));
+        required.push_back(AZ_CRC_CE("MultiplayerService"));
+        required.push_back(AZ_CRC_CE("AWSGameLiftServerService"));
+    }
+
+    void MultiplayerSampleAWSGameLiftServerSystemComponent::GetDependentServices(AZ::ComponentDescriptor::DependencyArrayType& dependent)
```

Comment:
Delete Dependent() method if it's not used

Reply:
I tend to feel as @mbalfour-amzn does, as someone who's not in the code base frequently enough to remember how to add these w/o help, but not _infrequently_ enough not to get irritated with having to dig up examples when I am knocking around in here 😆